src/flavors/pbdurham/config.json

```diff
@@ -1174,33 +1150,73 @@
     }
   },
   "survey": {
-    "submission_type": "comments",
-    "adding_enabled": false,
-    "show_responses": true,
-    "response_name": "_(comment)",
-    "response_plural_name": "_(comments)",
-    "action_text": "_(commented on)",
-    "anonymous_name": "_(Someone)",
-    "title": "_(Leave a Comment)",
-    "form_link_text": "_(Leave a Comment)",
-    "submit_btn_text": "_(Comment)",
-    "items": [
+    "comments": {
```

Comment:
Instead of having our "survey" config represent our comments, we now have "survey.comments" and "survey.forms" to represent the two types of surveys in the app.